tests/integration/test_database.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from rebirthdb.errors import ReqlRuntimeError
+from tests.helpers import IntegrationTestCaseBase, INTEGRATION_TEST_DB
+
+
+@pytest.mark.integration
+class TestCursorFor(IntegrationTestCaseBase):
```

Comment:
Looks like a copy/paste. This could match file name as `TestDatabase` or `TestCursorDB`. Not a blocking issue.